crates/config/src/providers/remappings.rs

```diff
@@ -196,7 +196,7 @@ impl RemappingsProvider<'_> {
     fn lib_foundry_toml_remappings(&self) -> impl Iterator<Item = Remapping> + '_ {
         self.lib_paths
             .iter()
-            .map(|p| self.root.join(p))
+            .map(|p| if p.is_absolute() { self.root.join("lib") } else { self.root.join(p) })
```

Comment:
if this is absolute, should we just use `p`? because if absolute this would exclude p entirely?

Reply:
This is an optional solution because we don't know the actual library path in the current directory, so we need to provide a default path.
My question is: Should we allow setting an absolute library path? If yes, there should be a default library path.

Reply:
I see, I'm leaning towards not allowing/supporting absolute lib paths, so I think the current default to using lib makes sense

Reply:
Agree, using the default library path can make most libraries work effectively.
If the solution is acceptable, I will add some tests for it.